spec/services/healthcheck_spec.rb

```diff
@@ -29,6 +30,16 @@
     allow(GlimrApiClient::Available).to receive(:call).and_return(glimr_available)
     allow(ActiveRecord::Base).to receive(:connection).and_return(double)
     allow(MojFileUploaderApiClient::Status).to receive(:new).and_return(uploader_client)
+    allow_any_instance_of(described_class).to receive(:`).with('git rev-parse HEAD').and_return('ABC123')
+  end
+
+  describe '.version' do
+    # Necessary evil for coverage purposes.
+    it 'calls `git rev-parse HEAD`' do
+      # See above
+      expect_any_instance_of(described_class).to receive(:`).with('git rev-parse HEAD').and_return('ABC123')
```

Comment:
Can we use `system(...)` (`Kernel.system`) instead?

Would be a bit cleaner in the implementation IMHO, and lead to less weird stubbing.